Algorithms.Tests/Search/Substring/RabinKarpTests.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using Algorithms.Search.Substring;
+using NUnit.Framework;
+
+namespace Algorithms.Tests.Search.Substring
+{
+    public class RabinKarpTest
+    {
+
+        [TestCase("HelloImATestcaseAndIWillPass", "Testcase", new int[] {8})]
+        [TestCase("HelloImATestcaseAndImCaseSensitiv", "TestCase", new int[] {})]
+        [TestCase("HelloImATestcaseAndIHaveTwoOccurrencesOfTestcase", "Testcase", new int[] {8, 40})]
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6278031)